debugger/src/debug_adapter/dap_adapter.rs

```diff
@@ -1632,7 +1640,15 @@ impl<P: ProtocolAdapter> DebugAdapter<P> {
                             named_variables: None,
                             presentation_hint: None, // TODO: Implement hint as Hex for registers
                             type_: Some(format!("{}", VariableName::RegistersRoot)),
-                            value: format!("{:#010x}", register_value),
+                            value: format!(
```

Comment:
This is a nice touch, although there are a LOT of other places in the code where format is hardcoded for `{:#010x}` ... we will have to clean that up at some point.

Reply:
bors r+